lib/rules/require-computed-macros.js

```diff
@@ -161,71 +172,116 @@ module.exports = {
 
   create(context) {
     function reportSingleArg(nodeComputedProperty, nodeWithThisExpression, macro) {
-      context.report({
-        node: nodeComputedProperty,
-        message: makeErrorMessage(macro),
-        fix(fixer) {
-          const text = propertyGetterUtils.nodeToDependentKey(nodeWithThisExpression, context);
-          return fixer.replaceText(
-            nodeComputedProperty,
-            makeFix(nodeComputedProperty, macro, `'${text}'`)
-          );
-        },
-      });
+      const text = propertyGetterUtils.nodeToDependentKey(nodeWithThisExpression, context);
+      const isNotFixable = detectCircularKey(nodeComputedProperty, [text]);
+
+      if (isNotFixable) {
+        context.report({
```

Comment:
@bmish I did that initially but it is linted against, I also suspect it's not good practice since it might cause eslint to report the violation as fixable when it is not.

Reply:
It's not linted against, we actually have some examples of autofixers with `return null` in our codebase. And it's not bad practice, in fact it's common practice to have autofixers which ignore certain situations, and this is how you do it. I can assure you it will result in much simpler code :)

Reply:
this was the lint rule it triggered https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/master/docs/rules/fixer-return.md

Reply:
Using `return null` won't trigger that rule. `return null` indicates that you don't want to autofix in a particular situation.